assets/js/hooks/metamask.js

```diff
@@ -0,0 +1,56 @@
+import {ethers} from "../../vendor/ethers"
+
+const web3Provider = new ethers.providers.Web3Provider(window.ethereum)
+
+function init() {
+    // Check that the web page is run in a secure context,
+    // as otherwise MetaMask won't be available
+    if (location.hostname !== "localhost" && location.protocol !== 'https:') {
+        console.log("FAILING AT HTTPS")
+    }
+}
+
+export const Metamask = {
+    mounted() {
+        let signer = web3Provider.getSigner()
+
+        window.addEventListener('load', async () => {
+            init();
+            let address
```

Comment:
I'm struggling to understand why this is needed. It seems it's nothing is ever assigned to this variable

Reply:
I forgot this here when cleaning up the code. You're right, it's doing nothing...